static/tax_required_false/one_person_case/client_request.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
same issue here with the "tax_required_false" directory. Also, don't you have earlier files that also relate to tax requirements and incarceration anyway?